governance/working groups/index.md

```diff
@@ -0,0 +1,70 @@
+# Working groups
+
+For the UK TRE Community, a Working Group (WG) is the main method through which community outputs are created.
+
+WGs can be set up by any member of the community, and are focused on producing specific, tangible outputs within a given time period, and are modelled off the pre-existing concept of Working Groups found in communities like [DARE UK](https://dareuk.org.uk/dare-uk-launches-dynamic-collaborative-communities-invites-proposals-for-new-groups/), the [Research Data Alliance (RDA)](https://www.rd-alliance.org/groups/creating-and-managing-rda-groups/creating-or-joining-rda-working-group.html), the [W3C](https://www.w3.org/2017/Process-20170301/#GAGeneral) and more.
+
+There are two distinct processes for those considering creating WGs to consider:
+1. The establishment of the WG
+2. The community endorsement of the WG outputs
+
+As the UK TRE community's primary focus is to act as a signposting and convening body for the TRE space in the UK and beyond, we have made a conscious effort to separate out the questions of:
+- Work that is happening within the community (WG establishment)
+- Outputs/resources for which there is community consensus and endorsement (community endorsement)
+
+Therefore it is important to note that establishing a WG with the UK TRE community **does not** imply any outputs of the WG are endorsed by the UK TRE Community.
+
+
+## Process
+### Establishing a Working group
+
+1. Member(s) suggesting a working group fill in the [Working Group Charter](working-group-charter.md).
+2. The Charter is reviewed by the [Steering Group]() to ensure it:
```

Comment:
Is the steering group the same as the (possible renamed) CMWG, as opposed to a WG steering group?

Reply:
I am updating this to be clearer - the CMWG and SG are different - the CMWG is responsible for quickly checking whether the proposal fits with community principles and isn't being done anywhere else, the SG comes in later for formal approval/rejection (this will make more sense after the next commit)